src/lib/components/SearchPage.svelte

```diff
@@ -28,6 +29,13 @@
 			const state = window.history.state;
 
 			window.history.replaceState(state, '', url);
+			
+			const path = window.location.pathname;
+			if (path.startsWith('/search')) {
```

Comment:
Nice, but you didn't take into consideration the base, import these at the top 

```ts
import { page } from '$app/stores';
import { base } from '$app/paths';
```

and replace this line with :
```ts
if ($page.url.pathname.startsWith(`${base}/search`)) {
```

and we should be good to go !